mfx/optimize/xopt_scans.py

```diff
@@ -173,33 +180,68 @@ def get_xopt_obj(
 
     Parameters
     ----------
-    device_type: str
+    device_type : str
         One of "yag" or "wave8"
     location : str
-        One of "xcs1", "dg1", "dg2", "ip"
+        One of "xcs1", "dg1", "dg2", "ip" # TODO: should IP be added to Diagnostics?
     goal : float
         Either the wave8 xpos to aim for, or the x coordinate to aim for on a yag.
     mirror_nominal : float
         The starting mirror pitch position and midpoint of the optimization search.
     search_delta : float
         How far +/- we check away from the mirror nominal pitch position
+    wave8_max_value: float, optional
+        Constraint on maximum wave8 xpos for data to be valid
     yag_size_min : float, optional
-        Constraint on minimum yag spot size for data to be valid
+        Constraint on minimum yag spot size (RMS) for data to be valid
     yag_size_max : float, optional
-        Constraint on maximum yag spot size for data to be valid
+        Constraint on maximum yag spot size (RMS) for data to be valid
     yag_intensity_min : float, optional
         Constraint on minimum yag total intensity count for data to be valid
     yag_intensity_max : float, optional
         Constraint on maximum yag total intensity count for data to be valid
+    centroid_x_min : float, optional
+        Constraint on minimum centroid x value for data to be valid
+    centroid_x_max : float, optional
+        Constraint on maximum centroid x value for data to be valid
+    centroid_y_min : float, optional
+    Constraint on minimum centroid y value for data to be valid
```

Comment:
Docs spacing nitpick
```suggestion
        Constraint on minimum centroid y value for data to be valid
```